upgrade-scripts/postgresql/pim-7.73-to-7.74.sql

```diff
@@ -0,0 +1,2 @@
+alter table if exists migration_reports
```

Comment:
right name should be `pim-7.71-to-7.73.sql` (there won't be any 7.74)

Reply:
Done